<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DllProjectDialog</class>
 <widget class="QDialog" name="DllProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DllProjectDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QPushButton" name="pushButton_Test">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Type">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Value">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>返回值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入值：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
